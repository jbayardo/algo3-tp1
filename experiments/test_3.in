#grafo completo

abc a bc;b ac;c ab
abcd a bcd;b acd;c abd;d abc
abcde a bcde;b acde;c abde;d abce;e abcd
abcdefgh a bcdefgh;b acdefgh;c abdefgh;d abcefgh;e abcdfgh;f abcdegh;g abcdefh;h abcdefg

#1 rel + 4 conexas, sin ciclos
abcd a b;b a;c d;d c
abcdef a b;b a;c d;d c;e f;f e
abcdefgh a b;b a;c d;d c;e f;f e;g h;h g

#1 rel + 1 ciclo + 1 conexa
abcd a db;b ac;c bd;d ca
abcdef a fb;b ac;c bd;d ce;e df;f ea
abcdefgh a hb;b ac;c bd;d ce;e df;f ge;g hf;h ga

#2 rel + 2 ciclos + 2 comp conexas
abcdefgh a bd;b ac;c bd;d ac;e hf;f eg;g hf;h eg

#3 rel + 2 conexas, sin ciclos
abcde a b;b ac;c b;d e;e d
abcdefgh a b;b a;c d;d ce;e fgh;f e;g e;h e
abcdefgh a b;b adc;c b;d b;e hf;f eg;g f;h e

#2 rel + 3 comp conexas
abcdefghi a b;b ac;c b;d i;i hd;h i;e f;f eg;g f

#arbol binario
abcdefgh a bc;b ade;c afg;d b;e b;f c;g ch;h g

#estrella, 2 estrellas
abcdefgh a bcdefgh;b a;c a;d a;e a;f a;g a;h a







abcde a bcde;b acde;c abde;d abce;e abcd
abcdefghi a bcdefghi;b acdefghi;c abdefghi;d abcefghi;e abcdfghi;f abcdeghi;g abcdefhi;h abcdefgi;i abcdefgh
