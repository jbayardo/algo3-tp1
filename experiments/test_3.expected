#grafo completo
1 abc
2 abcd
2 abcde
4 abcdefgh

#1 rel + 4 conexas, sin ciclos
1 abcd
1 abcdef
1 abcdefgh

#1 rel + 1 ciclo + 1 conexa
1 abcd
1 abcdef
1 abcdefgh

#2 rel + 2 ciclos + 2 comp conexas
3 abcdefgh

#3 rel + 2 conexas, sin ciclos
1 abcde
2 abcdfegh
2 abcdgfeh

#2 rel + 3 comp conexas
1 abcdihefg

#arbol binario
2 abdehgcf

#estrella
4 abcdefgh

2 abcde
4 abcdefghi